druid/examples/sub_window.rs

```diff
@@ -326,8 +326,14 @@ impl<W: Widget<bool>> Controller<bool, W> for CancelClose {
 }
 
 fn build_root_widget() -> impl Widget<HelloState> {
+    let env = {
+        let mut env = Env::new();
+        env.set(theme::TEXT_COLOR, env.get(theme::PRIMARY_LIGHT));
+        env
```

Comment:
the easier way to do this is,

```rust
let env = Env::new() .adding(theme::TEXT_COLOR, Color::SOME_COLOR);
```

I'm not sure why we're trying to get a value out of the new env, here, it should be empty?

Reply:
Oh ok, I was mostly trying to keep the example the same.

Reply:
Oh I see, I had just copied it without thinking about what it was actually doing.